src/payload_size.rs

```diff
@@ -43,9 +44,11 @@ mod payload_size_tests {
     }
     #[test]
     fn test_exact_align_overhead() {
-        // Scenario: 4 KB payload split over 1024 bytes content forces a fifth chunk of overhead
+        // Scenario: 4 KB payload split over 1024 bytes content forces a fifth chunk of
+        // overhead
         assert_eq!(number_chunks_overhead(4 * 1024, 1024), 5);
-        // Scenario: 120 bytes payload split over 30 bytes content forces a fifth chunk too
+        // Scenario: 120 bytes payload split over 30 bytes content forces a fifth chunk
+        // too
```

Comment:
Yeah no, the one-line-ness of the scenario must be respected, so my `show-gherkin` aliases (in Emacs, but grep version exists) works still.

I'll clear it up by shortening the scenario.

Reply:
in that case you might want to add a flag or change the configuration of rustfmt to ignore these. pre-commit did this one